src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/ListViewModel.cs

```diff
@@ -248,6 +266,31 @@ private void InitializeItemsTask(CancellationToken ct)
         }
     }
 
+    public void UpdateGroupsIfNeeded()
+    {
+        Groups.Clear();
```

Comment:
okay so I think this is conna cause flickering on pages that update. I'll try to whip up a sample extension that repros this.

But you can kinda see it on the main page if you have a bunch of extensions enabled - as each one gets loaded, we RaiseItemsChanged on the main page, which then causes us to get here in the ViewModel, and blow away all the old groups and create new ones. It causes just like, one frame of flickering.

I think we need on instead try to see if we already had a group for that section tag, and do the `InPlaceUpdateList` thing on that existing group. That should suppress the flickering.